src/xmpp/message.c

```diff
@@ -1261,7 +1261,7 @@ _handle_muc_private_message(xmpp_stanza_t* const stanza)
     ProfMessage* message = message_init();
     message->type = PROF_MSG_TYPE_MUCPM;
 
-    const gchar* from = xmpp_stanza_get_from(stanza);
+    const char* from = xmpp_stanza_get_from(stanza);
```

Comment:
There are few more cases, I suggest to catch all of them, using regex: `gchar.+xmpp_`